src/lib.rs

```diff
@@ -200,6 +202,12 @@ feature! {
     pub mod spawn;
 }
 
+#[cfg(unix)]
```

Comment:
I've tested on ubuntu and archlinux also.

Perviously add `cfg(any(macos, linux))` and clippy notice me to make it `cfg(unix)`.